src/components/Sidebar.tsx

```diff
@@ -318,7 +318,7 @@ class Sidebar extends PureComponent<Props & RouteComponentProps, State> {
       history.push(routes.SEND);
     });
 
-    // Import Private Keys
+    // Import a Private Key
```

Comment:
This won't change the Electron menu label, as suggested in commit message. This changes only the code comment.

To change the label, update `public/electron.js` file.